src/ui/popup/App.tsx

```diff
@@ -362,10 +362,23 @@ class App extends React.Component<PopupAppComponentProps, InitialState> {
             iconName="cog"
             className="btn-info"
             onClick={() => {
-              browser.tabs.create({
-                index: tab.index + 1,
-                url: '/settings/settings.html#tabSettings',
-              });
+              if (
+                cache.browserDetect &&
```

Comment:
Optional: I feel like a lib function `isFirefox` and `isChrome` where you pass in the cache and return true or false would be useful here, since this browser detection would be used a lot.

Reply:
Done via c66c6ea